ansible/site.yml

```diff
@@ -1,5 +1,34 @@
 ---
+
+# Note: OS specific variables are included in
+# the order of the hierarchy of their fork...
+# The "lower" fork wins
+# Debian <
+#  Ubuntu <
+#   Raspbian
+#   Armian
+
 - hosts: all
+  pre_tasks:
+    - name: Include OS family variables
+      include_vars: "group_vars/{{ ansible_os_family|lower }}"
+    - name: Include Distribution variables
+      include_vars: "group_vars/{{ ansible_distribution|lower }}"
+      when: ansible_os_family != ansible_distribution
+    - name: Include Target specific variables
+      include_vars: "group_vars/{{ ansible_lsb['id']|lower }}"
+      when: ansible_lsb["id"] != ansible_distribution
```

Comment:
Is there any harm in dropping the `when:` and unconditionally importing? (not sure whether it produces warnings about redefinition of variables)